packages/hint/src/lib/cli/analyze.ts

```diff
@@ -188,8 +188,19 @@ const loadUserConfig = async (actions?: CLIOptions): Promise<UserConfig> => {
         userConfig = getDefaultConfiguration();
     }
 
-    // If language is not in the config file, get the language configure in the OS.
-    userConfig.language = userConfig.language || await osLocale();
+    // If language is not in the config file or CLI options, get the language configure in the OS.
+    if (actions && actions.language) {
```

Comment:
Can you move all of this to a method `getLanguage`?

Reply:
yeah moving it. Forgot in previous commit.

Reply:
@sarvaje move language logic to `getLanguage`. Please have a look. `userConfig` is made optional in case we use `getLanguage` function somewhere else in file. So it will give `osLocale` in case both `userConfig` and `actions` are `undefined`